.travis.yml

```diff
@@ -35,6 +35,8 @@ env:
   global:
     - REPO="git@github.com:RedHatInsights/insights-chrome-build"
     - REPO_DIR="insights-chrome-build"
+    - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
+    - SENTRY_AUTH_KEY=${SENTRY_AUTH_KEY}
```

Comment:
We do not use Travis anymore to publish our assets. This has to be set up in the vault.

Reply:
Thank you. I do not know what is vault. Can you some guidance on what is it and how I can provide the secret to node env so that I can fix this?

Reply:
I think you won't be able to do it because it is the platex store. If you PM me the secret, I will ensure it will be added to the vault store.